### PR TITLE
Testcases for template

### PR DESCRIPTION
**Pull request by dsteinkopf**
add tests for template and add docu for path in json

### CODE REVIEW

File: dbus_service.py [hunk 1]
```diff
@@ -171,10 +171,10 @@ def _get_config():
         return config
 
     @staticmethod
-    def get_processed_meter_value(meter_data: dict, value: str, default_value: any, factor: int = 1) -> any:
+    def get_processed_meter_value(meter_data: dict, path_to_value, default_value: any, factor: int = 1) -> any:
```
Comment (0x7878) [inline dbus_service.py:174]: Why did you remove the type?

```suggestion
    def get_processed_meter_value(meter_data: dict, path_to_value: str, default_value: any, factor: int = 1) -> any:
```

File: dbus_service.py [hunk 2]
```diff
@@ -484,7 +484,7 @@ def fetch_url(self, url, try_number=1):
             else:
                 raise
 
-    def _get_data(self):
+    def _get_data(self) -> dict:
```
Comment (0x7878) [inline dbus_service.py:487]: Very nice!

**Review by 0x7878 — APPROVED** [review]
Cool that you wrote a test for it too. I assume it works as intended.  It is your decision whether or not to include the type in the review.

File: dbus_service.py (hunk 1 above)
Comment (dsteinkopf) [reply dbus_service.py:174]: because this is actually not a string - it is an array of strings. But not always... So the "str" here is more confusing than helping

We should clean this up...
